VENTokenField/VENTokenField.m

```diff
@@ -338,6 +343,12 @@ - (void)layoutTokensWithCurrentX:(CGFloat *)currentX currentY:(CGFloat *)current
         
         [self.tokens addObject:token];
 
+        // we need this adjustment to horizontally align the tokens with the toLabel
+        if (i == 0) {
+            CGFloat toLabelMidY = self.toLabel.frame.origin.y + self.toLabel.frame.size.height/2;
+            *currentY = toLabelMidY - token.frame.size.height/2;
+        }
```

Comment:
Before this change, it was setting the currentY to - 
`*currentY += token.frame.size.height;`
This makes sure we horizontally align the token to the label.